subsys/bluetooth/audio/tbs_client.c

```diff
@@ -1931,6 +1937,7 @@ int bt_tbs_client_read_current_calls(struct bt_conn *conn, uint8_t inst_index)
 		return -EINVAL;
 	}
 
+	initialize_net_buf_read_buffer(inst);
```

Comment:
Please add a comment why this is only used for the `read_current_calls_cb` and `read_call_state_cb` (the answer of course is that those are characteristic that may be larger than the minimum MTU). 

That being said, it would appear that we also should use it for any of the string characteristics that contain strings, i.e. Bearer Name, supported URIs, Incoming call target URI, Incoming Call and Call Friendly name.

Reply:
Maybe we should add a function that returns true or false depending on whether the characteristic may be > 23 octets?

Reply:
> That being said, it would appear that we also should use it for any of the string characteristics that contain strings, i.e. Bearer Name, supported URIs, Incoming call target URI, Incoming Call and Call Friendly name.

Makes sense. I can add code to use the same net_buf handling for the strings as we do for the other two possible long values.
What would the function which checks for the 23 octets be used for?

Reply:
> > That being said, it would appear that we also should use it for any of the string characteristics that contain strings, i.e. Bearer Name, supported URIs, Incoming call target URI, Incoming Call and Call Friendly name.
> 
> Makes sense. I can add code to use the same net_buf handling for the strings as we do for the other two possible long values. What would the function which checks for the 23 octets be used for?

Nothing really; at the time of writing I must have been thinking like
```
if (get_char_max_len() > 23) {
    initialize_net_buf_read_buffer(inst);
}
```

But it doesn't really make sense to put that as a function :P